backend/fleet_management/signals.py

```diff
@@ -1,39 +0,0 @@
-from django import dispatch
```

Comment:
What does this change (and the ones above) have in common with adding flake8?

Reply:
After linter was updated, it detected dead code. It wouldn't pass without a cleanup.

Reply:
@arturtamborski

Reply:
OK, but I still don't think that this code should be removed.

@magul / @monikakrawczak can you confirm the current situation with e-mails? Are we planning to send them out eventually? I see that we have a template mail and the code looks like it was used but was commented out due to production issues mentioned here: https://github.com/CodeForPoznan/pah-fm/pull/209 
Did anyone tried to take a look into it and fix it?

Anyway, I'd opt out for commenting this function out or adding this file to ignore list in flake8 instead of removing (what looks to be) a good, but unused code.

Reply:
Discussed this offline with @magul, seems like it is in fact not needed, so I'm giving a green light on this.